pkg/security/apparmor/validate_test.go

```diff
@@ -73,7 +74,7 @@ func TestValidateProfile(t *testing.T) {
 		if test.expectValid {
 			assert.NoError(t, err, "Profile %s should be valid", test.profile)
 		} else {
-			assert.Error(t, err, "Profile %s should not be valid", test.profile)
+			assert.Error(t, err, fmt.Sprintf("Profile %s should not be valid", test.profile))
```

Comment:
It's a little odd to fix this line and not the NoError line right above this. Is this really necessary for go 1.7 to work? Should it be in this PR?